gradle-scripts/dependencies.gradle

```diff
@@ -0,0 +1,13 @@
+dependencies {
+    implementation "com.commercetools.sdk.jvm.core:commercetools-models:${commercetoolsJvmSdkVersion}"
+    implementation "com.commercetools.sdk.jvm.core:commercetools-java-client:${commercetoolsJvmSdkVersion}"
+    implementation "com.commercetools.sdk.jvm.core:commercetools-convenience:${commercetoolsJvmSdkVersion}"
+    implementation group: 'javax.mail', name: 'javax.mail-api', version: '1.6.0'
+    implementation group: 'org.apache.commons', name: 'commons-email', version: '1.3.1'
+    implementation group: 'commons-io', name: 'commons-io', version: '2.5'
```

Comment:
Please use extensions for saving the dependency versions in the extensions.gradle like the other dependencies in the same file

Reply:
outdated